.vale/styles/Silverstripe-cms/Repetition.yml

```diff
@@ -0,0 +1,12 @@
+---
+# Error: gitlab.Repetition
+#
+# Checks for duplicate words, like `the the` or `and and`.
+#
+# https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/.vale/gitlab/Repetition.yml
+extends: repetition
+message: '"%s" is repeated.'
+level: error
+alpha: true
+tokens:
+  - '[^\s]+'
```

Comment:
Add blank at end of file

Reply:
Marking resolved since this is now out of scope.